trading_calendars/xtks_holidays.py

```diff
@@ -11,28 +11,24 @@
 )
 
 from .common_holidays import new_years_day
-from .trading_calendar import SATURDAY, SUNDAY
+from .trading_calendar import SUNDAY
 
 
-def weekend_to_tuesday(dt):
+def sunday_to_tuesday(dt):
     """
     If holiday falls on Sunday or Saturday, use Tuesday instead.
     """
     if dt.weekday() == SUNDAY:
         return dt + timedelta(2)
-    elif dt.weekday() == SATURDAY:
-        return dt + timedelta(3)
     return dt
 
 
-def weekend_to_wednesday(dt):
+def sunday_to_wednesday(dt):
     """
     If holiday falls on Sunday or Saturday, use Wednesday instead.
```

Comment:
Could you update this docstring to reflect your update?

Reply:
all done!